lib/src/arithmetic/arithmetic_utils.dart

```diff
@@ -0,0 +1,112 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// floating_point_test.dart
+// Tests of Floating Point stuff
+//
+// 2024 August 30
+// Author: Desmond A Kirkpatrick <desmond.a.kirkpatrick@intel.com
+
+// ignore_for_file: avoid_print
+
+import 'package:rohd/rohd.dart';
+import 'package:rohd_hcl/rohd_hcl.dart';
+
+/// Helper evaluation methods for printing aligned arithmetic bitvectors.
+extension NumericVector on LogicValue {
+  /// Print aligned bitvector with an optional header.
+  /// [name] is printed at the LHS of the line, trimmed by [prefix].
+  /// [prefix] is the distance from the margin bebore the vector is printed.
+  /// You can align with longer bitvectors by stating the length [align].
+  /// [lowLimit] will trim the vector below this bit position.
+  /// You can insert a separator [sepChar] at position [sep].
+  /// A header can be printed by setting [header] to true.
+  /// Markdown format can be produced by setting [markDown] to true.
```

Comment:
Might be nice to include an example of what this looks like in the doc comment?  You can wrap it in "```" to display it as code

Reply:
I was wondering what to do about these debug routines that I use.  I call them 'evaluate', but they are really introspection into the component or printing live values.  Where should we put these in both code and documentation?
I put some of them in the multiplier_components.md, for example.  I can put an example there.

Reply:
Sure that works too, to clarify my suggestion it was something like this:
https://github.com/intel/rohd/blob/3e2d24a78adc7579fb7d4a99452d2105a2e70b65/lib/src/signals/logic.dart#L650-L676

Which then looks like this in generated API docs:
https://intel.github.io/rohd/rohd/Logic/getRange.html

Reply:
I need to do this much more often.  There are many cases an invocation example is the best documentation.  I put something in the documentation, but I will work toward this example.